content/en/apps/guides/hosting/monitoring/setup.md

```diff
@@ -101,6 +101,44 @@ When you see a new version in the [GitHub repository](https://github.com/medic/c
 
 ### Additional Configuration
 
+#### couch2pg Data
+
+With the [release of 1.1.0](https://github.com/medic/cht-monitoring/releases/tag/1.1.0), CMA now supports easily ingesting [couch2pg]({{< relref "apps/tutorials/couch2pg-setup" >}}) data read in from a Postgres database.  While 1.1.0 only currently supports the couchd2pg backlog metric, more queries will be added over time.
```

Comment:
```suggestion
With the [release of 1.1.0](https://github.com/medic/cht-monitoring/releases/tag/1.1.0), CMA now supports easily ingesting [couch2pg]({{< relref "apps/tutorials/couch2pg-setup" >}}) data read in from a Postgres database.
```

IMHO it is okay not to mention this. I expect we will add more queries very soon and we would have to remember to come back here and remove this then....